pkg/modules/observability.go

```diff
@@ -799,6 +822,69 @@ func getNewAuthSecretName(driverType csmv1.DriverType, secretName string) string
 	return fmt.Sprintf("%s-%s", driverType, secretName)
 }
 
+// getIssuerCertServiceObs - gets cert manager issuer and certificate manifest for observability
+func getIssuerCertServiceObs(op utils.OperatorConfig, obs csmv1.Module, componentName string) (string, error) {
+	yamlString := ""
+	certificate := ""
+	privateKey := ""
+	certificatePath := ""
+
+	for _, component := range obs.Components {
+		if component.Name == componentName {
+			certificate = component.Certificate
+			privateKey = component.PrivateKey
+		}
+	}
+
+	// If we have at least one of the certificate or privateKey fields filled in, we assume the customer is trying to use a custom cert.
+	// Otherwise, we give them the self-signed cert.
+	if certificate != "" || privateKey != "" {
```

Comment:
is it possible that a customer might want to use their own cert instead of one deployed by operator?

Reply:
I'm not aware of that being a valid use-case, @tdawe can you comment?

Reply:
@JacobGros I think if they want to they could always replace the self-signed certs with their own certs

Reply:
after deploying

Reply:
@tdawe but this is just them passing the certificate and private key blocks, right? not a whole different cert issuer etc?